setup.py

```diff
@@ -39,6 +39,7 @@
         "netcdf4",
         "numpy",
         "rasterio",
+        "requests",
         "shapely",
         "tqdm",
```

Comment:
Hello @asjohnston-asf , I noticed a couple of requirements which are out-of-sync between `setup.py` and the `environment.yml`

Specifically `asf_search` should be `asf_search>=5.0.0` and `dem_stitcher` should be `dem_stitcher>=2.5.8`

Reply:
Here's a PR to fix that: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/215